### PR TITLE
[*] integ test data fix

### PR DESCRIPTION
**Pull request by dusanDrabik**
https://decentplatform.atlassian.net/browse/DJS-21

### CODE REVIEW

File: src/test/client/integration/content.test.js [hunk 1]
```diff
@@ -33,30 +33,29 @@ describe('(client/integration) Content fetch', () => {
...
-            .then(res => {
-                expect(res).a('string');
-                done();
-            })
-            .catch(err => {
-                expect(err).to.be.a('array');
-                done();
-            });
-    }).timeout(5000);
+    // it('restore content keys', (done) => {
```
Comment (hidden) [inline src/test/client/integration/content.test.js]: Why is this block commented? Shouldn't it be removed?

File: src/test/server/integration/content.test.ts [hunk 2]
```diff
@@ -45,30 +45,29 @@ describe('(server/integration) Content fetch', () => {
...
-            .then(res => {
-                expect(res).a('string');
-                done();
-            })
-            .catch(err => {
-                expect(err).to.be.a('array');
-                done();
-            });
-    }).timeout(5000);
+    // it('restore content keys', (done) => {
```
Comment (hidden) [inline src/test/server/integration/content.test.ts]: Same here - why comments?

Comment (dusanDrabik) [reply src/test/server/integration/content.test.ts]: That is integration test for content restoration. However, due to stage testnet we do not have content, we cannot restore content keys.

Comment (hidden) [reply src/test/server/integration/content.test.ts]: There are two options - explain this in at the beginning of comment section or remove test.

**Comment by dusanDrabik** [comment]
Commented tests description added

**Pull request merged**